sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -2277,8 +2277,6 @@ public void testNullEmptyStringEquality() throws Exception
   @Test
   public void testNullLongFilter() throws Exception
   {
-    // bug https://github.com/apache/incubator-druid/issues/9062 prevents this test from passing
-    skipVectorize();
```

Comment:
Should we look for any uses of the this and see evaluate if they should be removed?